torch/optim/adamw.py

```diff
@@ -1,114 +0,0 @@
-import math
-import torch
-from .optimizer import Optimizer
-
-
-class AdamW(Optimizer):
```

Comment:
`AdamW` did not disappear, but can be found in `optim.optimizerw` (which should not change the interface in any way).

Reply:
But the path to it changed, right?

Reply:
I would argue that "officially" nothing changed, since only `torch.optim` is exposed and the submodules are internal (deleted in `optim.__init__`). Therefore I would consider imports of the form 
```
from torch.optim.adamw import AdamW
from torch.optim import adamw
``` 
unsupported behaviour, since something like 
```
import torch.optim
torch.optim.adamw
```
will throw an `AttributeError`.

Reply:
Fair point indeed :)